packages/back-end/types/experiment.d.ts

```diff
@@ -36,7 +36,20 @@ export type DecisionFrameworkExperimentRecommendationStatus =
   | { status: "rollback-now" }
   | { status: "ready-for-review" };
 
-export type DecisionFrameworkData = DecisionFrameworkExperimentRecommendationStatus & {
+export type ExperimentUnhealthyData = {
+  // if key exists, the status is unhealthy
+  srm?: boolean;
+  multipleExposures?: { rawDecimal: number; totalUsers: number };
+  lowPowered?: boolean;
+};
+
+export type RunningExperimentStatus =
+  | DecisionFrameworkExperimentRecommendationStatus
+  | { status: "no-data" }
+  | { status: "unhealthy"; unhealthyData: ExperimentUnhealthyData }
+  | { status: "before-min" };
```

Comment:
suggestion: Can we make this more explicit? `before-min-duration` maybe?

Reply:
Of course, done.